### PR TITLE
339 UI updates to rooms

### LINKED ISSUE #339: Rooms UI update
The rooms page shows a list of rooms as well as the form to create a new room. Neither of these match the mockups.
### What needs to be done
 
Update the vue to match the mockups. 
If changes are drastic, make sure tests are updated to match.

### PR DESCRIPTION
**Pull request by KeeganWoods1**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**
Please provide enough information so that others can review your pull request:

Update the UI in Rooms to more closely match the mockups.

### Test plan (required)
Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

For UI Rooms Task

### Closing issues
List all issues this PR resolves.
- `closes #339 ` 

### Pre-Merge checklist 
Complete these items before requesting reviews
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions 

### Sub-tasks Checklist
- [x] UI Implementation
- [x] UI Test
- [ ] Backend Implementation
- [ ] Backend Tests
- [ ] Integration Tests
- [ ] System Tests
- [ ] Documentation

### CODE REVIEW

File: resources/js/Pages/Admin/Rooms/RoomsList.vue [hunk 1]
```diff
@@ -1,275 +0,0 @@
...
-                                    <div class="text-md mx-3">
-                                    <button
-                                        class="cursor-pointer ml-6 text-sm text-blue-800 focus:outline-none"
-                                        @click="roomBeingUpdated = room"
-                                    >
-                                        Update
-                                    </button>
-                                </div>
-                                <div class="text-md mx-3">
-                                    <jet-dropdown-link :href="'/rooms/'+room.id+'/blackouts'">
```
Comment (Alex-Hayeur) [inline resources/js/Pages/Admin/Rooms/RoomsList.vue:68]: This table is currently missing the function to blackout rooms. The old RoomList page had a button in its dropdown leading admins to all active blackouts for the selected room, and allowed admins to create a new blackout. 

The new table should keep that, and hopefully could also put the 4 action button columns into one column as a dropdown to keep the list more compact.

**Review by Alex-Hayeur — CHANGES_REQUESTED** [review]
The table is looking good and better matches our approved UI mockups. But I have a comment on some missing functionality.

File: resources/js/Pages/Admin/Rooms/RoomsList.vue (hunk 1 above)
Comment (KeeganWoods1) [reply resources/js/Pages/Admin/Rooms/RoomsList.vue:68]: I just saw that this was missing, I will be turning the last columns of the table into a dropdown, thank Alex!

**Review by Alex-Hayeur — APPROVED** [review]
Looks good now, but for the sake of the others reviewing, can you please update the screenshot in the commit message at the top? So it shows the current UI.

**Review by danseremet — APPROVED** [review]
LGTM!

**Pull request merged**